packages/grafana-ui/src/components/PanelChrome/HoverWidget.tsx

```diff
@@ -0,0 +1,89 @@
+import { css } from '@emotion/css';
+import classnames from 'classnames';
+import React, { ReactElement, useCallback, useRef } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+import { Icon } from '../Icon/Icon';
+
+import { PanelMenu } from './PanelMenu';
+
+interface Props {
+  menu: ReactElement | (() => ReactElement);
+  title?: string;
+  offset?: number;
+}
+
+export function HoverWidget({ menu, title, offset = -32 }: Props) {
+  const styles = useStyles2(getStyles);
+  const draggableRef = useRef<HTMLDivElement>(null);
+
+  // Capture the pointer to keep the widget visible while dragging
+  const onPointerDown = useCallback((e: React.PointerEvent<HTMLDivElement>) => {
+    draggableRef.current?.setPointerCapture(e.pointerId);
+  }, []);
+
+  const onPointerUp = useCallback((e: React.PointerEvent<HTMLDivElement>) => {
+    draggableRef.current?.releasePointerCapture(e.pointerId);
+  }, []);
+
+  return (
+    <div className={classnames(styles.container, 'show-on-hover')} style={{ top: `${offset}px` }}>
+      <div
+        className={classnames(styles.square, styles.draggable, 'grid-drag-handle')}
```

Comment:
Not every panel by default is draggable, just if the dragClass is passed down to PanelChrome.

Reply:
Ah, I see!